cli/src/commands/bookmark/create.rs

```diff
@@ -75,6 +76,15 @@ pub fn cmd_bookmark_create(
             ));
         }
     }
+    if target_commit.is_discardable(repo).is_ok_and(|b| b) {
+        writeln!(ui.warning_default(), "Target revision is empty.")?;
+        if args.revision.as_ref() == "@" {
```

Comment:
nit: better to compare `target_commit` with `wc_commit_id`?

Or just suggest `jj undo` which should be more reliable? If the bookmark is included in the immutable set, new working-copy commit will be created on top. `jj squash` wouldn't work in that case.